src/cgame/cg_rocket_draw.cpp

```diff
@@ -3631,6 +3671,7 @@ void CG_Rocket_RegisterElements()
 	RegisterElement<CrosshairIndicatorHudElement>( "crosshair_indicator" );
 	RegisterElement<CrosshairHudElement>( "crosshair" );
 	RegisterElement<SpeedGraphElement>( "speedometer" );
+	RegisterElement<PositionElement>( "position_indicator" );
```

Comment:
imo, we should call this "player_origin". Position indicator is not correct because it's not an indicator. And since this is more of a debug element, we can be more technical about the names.

Reply:
player_origin talks less to me, even if that's technically true.